Documentation/Signoffs/Head-On-Sensor.md

```diff
@@ -0,0 +1,60 @@
+# Head On Position Sensor
+## Functionality
+The purpose of the head on position sensor is to determine the target's location relative to the interceptor. Sensing equipment shall be attached to the front of the interceptor facing the test bed. The sensor will then read in data and make the necessary calculations to determine the X, Y, and Z position of the target. Higher accuracy is expected in the X and Y direction rather than Z, as to get an accurate reading on depth a time of flight sensor would be required with a high sampling rate, and this would be difficult and expensive given the test bed is only 6 feet long [1].
+## Constraints
+* C7: Design a sensor array that can detect approaching objects and relay their locations to the interceptor.
+  * This subsystem will act in conjunction with other position sensors to be able to effectively track the target in three deminsions.
+* C23: The head on position sensor shall have a maximum latency of 100ms and provide updates at least 10 times per second.
+* C24: The head on position  sensor shall be able to detect the target with an accuracy of at least 2 inches from six feet away.
```

Comment:
What does a 2 inch accuracy mean here? Is that in the x direction or y? Or is that distance?